packages/shared/src/mock/metadata.ts

```diff
@@ -89,6 +89,21 @@ export const mockMetadata = makeNxGroup(mockFilepath, 'NXroot', {
                 }),
               ],
             }),
+            makeNxGroup('old-style_signal_axes', 'NXdata', {
+              children: [
+                makeDataset('twoD', intType, [20, 41], {
+                  attributes: [
+                    makeIntAttr('signal', 1),
+                    makeStrAttr('axes', 'Y,X'),
```

Comment:
I would maybe use a colon here, as the comma is an alternative proposed by the spec and not the main suggested delimiter. Testing the comma seems less important.